auth/simple_test.go

```diff
@@ -150,6 +166,26 @@ func TestAuthenticateFailure(t *testing.T) {
 	require.Equal(t, "application/json", recorder.HeaderMap["Content-Type"][0])
 }
 
+func TestAuthenticateWithOriginatorID(t *testing.T) {
```

Comment:
Also add a test for the case where there is no originator provided?

Reply:
Yeah that's covered in the testcase above `TestAuthenticationFailure`.